src/main/java/com/hivemq/client/mqtt/mqtt5/message/subscribe/Mqtt5RetainHandling.java

```diff
@@ -27,8 +27,17 @@
  */
 public enum Mqtt5RetainHandling {
 
+    /**
+     * Send retained messages at the time of the subscribe.
+     */
     SEND,
+    /**
+     * Send retained messages at subscribe only if the subscription does not currently exist.
+     */
     SEND_IF_SUBSCRIPTION_DOES_NOT_EXIST,
+    /**
+     * Do not send retained messages at the time of the subscribe.
```

Comment:
Only the subscription exists. Otherwise it needs to be something such as the subscribe command.